docs/content/tools/collector/_index.md

```diff
@@ -0,0 +1,494 @@
+```
+
+Provide parameters via ConfigFile:
+
+```powershell
+.\1_Wara_Collector.ps1 -configFile config.txt
+```
+
+config.txt
+
+```text
+[tenantid]
+00000000-0000-0000-0000-000000000000
+
+[subscriptionIds]
+/subscriptions/00000000-0000-0000-0000-000000000000
+/subscriptions/33333333-3333-3333-3333-333333333333
+
+[resourceGroups]
+/subscriptions/55555555-5555-5555-5555-555555555555/resourceGroups/RG-1
+
+[tags]
+Criticality=~High
+Env=~Prod
+```
+
+**Important**: Note that since you are filtering Sub-5 by ResourceGroup, you do not pass Sub-5 in the Subscriptions parameter. In this scenario, the script will set as scope all resources in Sub-0 and Sub-3, and in Sub5/RG1, as long as they all have both Tags, Criticality and Env equal to the values that I defined, High and Prod. Resource that do meet these requirements will be put in a separate array called  "out-of-scope", you will see a Worksheet in the excel file with this name and all resources not validated by the tool.
+
+### Example 4
+
+The workload is hosted in three subscriptions (Sub-0, Sub-4 and Sub-5), but Sub4 and Sub-5 subscriptions are shared with other workloads and I want to not only filter them by ResourceGroups where the components of my workload are hosted but also I need to filter by two Tags called Criticality and Env, since all my subscriptions are hosting NonProduction resources too:
+
+```powershell
+.\1_wara_collector.ps1 -TenantID "00000000-0000-0000-0000-000000000000" -SubscriptionIds "/subscriptions/00000000-0000-0000-0000-000000000000" -ResourceGroups "/subscriptions/55555555-5555-5555-5555-555555555555/resourceGroups/RG-1","
+","/subscriptions/44444444-4444-4444-4444-444444444444/resourceGroups/RG-2" -Tags 'Criticality=~High','Env=~Prod'
+```
+
+Provide parameters via ConfigFile:
+
+```powershell
+.\Wara_Collector.ps1 -configFile config.txt
+```
+
+config.txt
+
+```text
+[tenantid]
+00000000-0000-0000-0000-000000000000
+
+[subscriptionIds]
+/subscriptions/00000000-0000-0000-0000-000000000000
+
+[resourceGroups]
+/subscriptions/55555555-5555-5555-5555-555555555555/resourceGroups/RG-1
+/subscriptions/44444444-4444-4444-4444-444444444444/resourceGroups/RG-2
+
+[tags]
+Criticality=~High
+Env=~Prod
+```
+
+**Note**: Multiple values do not have to be in the same subscription. You can specify multiple resource groups in unique subscriptions.
+
+**Note**: In a configuration file we separate multiple entries for a filter by new lines. Where as, from the command line we would pass multiple subscriptions or resource groups using the "string1","string2" pattern. The configuration file is useful for repeated runs, or numerous filters where it may be difficult to troubleshoot syntax in the command line.
```

Comment:
```suggestion
{{< hint type=note >}}  
In the configuration file, we separate multiple entries for a filter by new lines. Where as, from the command line we would pass multiple subscriptions or resource groups using the "string1","string2" pattern. The configuration file is useful for repeated runs, or numerous filters where it may be difficult to troubleshoot syntax in the command line.
{{< /hint >}}  
```